ui/app/src/components/poolList/SinglePool.vue

```diff
@@ -54,33 +61,44 @@ export default defineComponent({
       const t = toToken.value;
       return t.imageUrl;
     });
-    const toValue = computed(() => accountPool?.value?.pool.amounts[0].toFixed(DECIMALS));
+
+    const toTotalValue = computed(() => accountPool?.value?.pool.amounts[0].toFixed(DECIMALS));
```

Comment:
I am wondering with this number formatting does it make sense to throw this all in a function now or wait until later?

Reply:
From our call this morning, I think someone wanted to rework the concept of to/from in most places, maybe it is better left until that refactor?

Reply:
Fair enough

Reply:
This file can be simplified pretty majorly